api/swagger.yaml

```diff
@@ -3507,6 +3557,14 @@ components:
               allOf:
                 - description: Minimum sensor value allowed
                 - $ref: '#/components/schemas/valueWithUnit'
+          example:
+            sensorUpdateFrequencyConstraintType: range
+            minimum:
+              value: 2
+              unit: ea
```

Comment:
ea?

Reply:
Standaard voor `each` eenheid, maar dit moet eigenlijk een frequentie grootheid zijn. Hier is nog ruimte voor verbetering in de spec.